src/main/java/org/broadinstitute/hellbender/tools/funcotator/mafOutput/MafOutputRenderer.java

```diff
@@ -482,6 +493,106 @@ else if ( value.toLowerCase().startsWith("chr") ) {
+                                MafOutputRendererConstants.VariantClassificationMap.get(GencodeFuncotation.VariantClassification.INTRON.toString())
+                        },
+                        new String[]{
+                                GencodeFuncotation.VariantClassification.IN_FRAME_DEL.toString(),
+                                GencodeFuncotation.VariantClassification.IN_FRAME_INS.toString(),
+                                GencodeFuncotation.VariantClassification.FRAME_SHIFT_INS.toString(),
+                                GencodeFuncotation.VariantClassification.FRAME_SHIFT_DEL.toString(),
+                                GencodeFuncotation.VariantClassification.MISSENSE.toString(),
+                                GencodeFuncotation.VariantClassification.NONSENSE.toString(),
+                                GencodeFuncotation.VariantClassification.SILENT.toString(),
+                                GencodeFuncotation.VariantClassification.SPLICE_SITE.toString(),
+                                GencodeFuncotation.VariantClassification.DE_NOVO_START_IN_FRAME.toString(),
+                                GencodeFuncotation.VariantClassification.DE_NOVO_START_OUT_FRAME.toString(),
+                                GencodeFuncotation.VariantClassification.START_CODON_SNP.toString(),
+                                GencodeFuncotation.VariantClassification.START_CODON_INS.toString(),
+                                GencodeFuncotation.VariantClassification.START_CODON_DEL.toString(),
+                                GencodeFuncotation.VariantClassification.NONSTOP.toString(),
+                                GencodeFuncotation.VariantClassification.FIVE_PRIME_UTR.toString(),
+                                GencodeFuncotation.VariantClassification.THREE_PRIME_UTR.toString(),
+                                GencodeFuncotation.VariantClassification.FIVE_PRIME_FLANK.toString(),
+                                GencodeFuncotation.VariantClassification.INTRON.toString(),
+                        }
+                );
+
+                // Handle the RNA/LINCRNA case specially:
+                final String needle = MafOutputRendererConstants.VariantClassificationMap.get(GencodeFuncotation.VariantClassification.LINCRNA.toString());
+                int indx = replacement.indexOf(needle);
+                while ( indx != -1 ) {
+                    // make sure the previous letters are not LINC:
+                    if ( (indx <= 3 ) || ((indx > 3) && (!replacement.substring(indx - 4, indx).equals("LINC")))) {
+                        replacement = replacement.substring(0, indx) + GencodeFuncotation.VariantClassification.LINCRNA.toString() + replacement.substring(indx + needle.length());
+                        indx += needle.length();
+                    }
+                    else {
+                        ++indx;
+                    }
+
+                    indx = replacement.indexOf(needle, indx);
+                }
+
```

Comment:
extra whitespace.

Reply:
I prefer to have more space so the code is readable.